book/website/community-handbook/accessibility/accessibility-comms-platforms.md

```diff
@@ -0,0 +1,29 @@
+(ch-accessibility-comms-platforms)=
+# Access-centered support on Communication Platforms
+
+_The Turing Way_ uses a variety of platforms for communication (for example, social media platforms like X/Twitter, Linkedin, Mastodon) and collaboration (for example, Github, HackMD, Etherpad).
```

Comment:
```suggestion
_The Turing Way_ uses a variety of platforms for communication (for example, social media platforms like X/Twitter, Linkedin, Mastodon) and collaboration (for example, Github, HackMD, Etherpad, Miro, Excalibur).
```

Reply:
Adding comment here reviewing @RichardJActon's comments about visual tools! We can add more documentation about this in this documentation at another date!